pkg/btf/validation.go

```diff
@@ -145,7 +145,11 @@ func ValidateKprobeSpec(bspec *btf.Spec, call string, kspec *v1alpha1.KProbeSpec
 		// NB: this might change in different kernels so if we fail we treat it as a warning
 		prefix := "__x64_sys_"
 		if !strings.HasPrefix(call, prefix) {
-			return &ValidationWarn{s: fmt.Sprintf("could not get the function prototype for %s: arguments will not be verified", call)}
+			if strings.HasPrefix(call, "__arm64_sys_") {
```

Comment:
fyi: you have `pkg/arch::HasSyscallPrefix()` existing helper and other helpers if you need.